Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This test references SAT - does it need to be updated to also include Secure SAT?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UIUnitTests/Areas/Groups/Controllers/GroupControllerTests.cs#L499-L509

```csharp

        [Theory]
        [InlineData(eLookupGroupType.SingleacademyTrust, "Single-academy trust name", "Close this single-academy trust and mark as created in error", "Group_EditDetails_DynamicLabels_SingleacademyTrust")]
        [InlineData(eLookupGroupType.MultiacademyTrust, "Multi-academy trust name", "Close this multi-academy trust and mark as created in error", "Group_EditDetails_DynamicLabels_MultiacademyTrust")]
        [InlineData(eLookupGroupType.ChildrensCentresCollaboration, "Children's centres collaboration name", "", "Group_EditDetails_DynamicLabels_ChildrensCentresCollaboration")]
        [InlineData(eLookupGroupType.ChildrensCentresGroup, "Children's centres group name", "", "Group_EditDetails_DynamicLabels_ChildrensCentresGroup")]
        [InlineData(eLookupGroupType.Federation, "Federation name", "Close this federation and mark as created in error", "Group_EditDetails_DynamicLabels_Federation")]
        [InlineData(eLookupGroupType.Trust, "Foundation trust name", "", "Group_EditDetails_DynamicLabels_Trust")]
        [InlineData(eLookupGroupType.SchoolSponsor, "Academy sponsor name", "Close this academy sponsor and mark as created in error", "Group_EditDetails_DynamicLabels_SchoolSponsor")]
        public async Task Group_EditDetails_DynamicLabels(eLookupGroupType groupType, string groupNameLabelText,
            string closeAndMarkAsCreatedInErrorLabelText, string testName)
        {
```

Reply:
Catherine has added a new line to test this.  Test passed.